Casks/sunloginclient.rb

```diff
@@ -13,7 +13,7 @@
     strategy :header_match
     regex(/SunloginClient-(\d+(?:\.\d+)*)\.dmg/i)
```

Comment:
Prefer a group match in place of `-` in regex

https://docs.brew.sh/Brew-Livecheck#regex-guidelines
> Use [._-] in place of a period/underscore/hyphen between the software name and version in a file name. 

```suggestion
    regex(/SunloginClient[._-](\d+(?:\.\d+)*)\.dmg/i)
```